diff --git a/src/layers/transformer.cc b/src/layers/transformer.cc
index 8658175b..270141ff 100644
--- a/src/layers/transformer.cc
+++ b/src/layers/transformer.cc
@@ -309,9 +309,9 @@ namespace ctranslate2 {
                   scope + "/layer",
                   _num_heads,
                   model.get_flag_with_default(scope + "/pre_norm", true),
-                  model.get_enum_value<ops::ActivationType>(scope + "/activation"),
+                  model.get_enum_value<ops::ActivationType>(scope + "/activation")
 #ifdef CT2_USE_HIP
-                  _use_flash_attention,
+                  , _use_flash_attention
 #endif
       ))
       , _position_encoder(_layers.front()->get_self_attention().has_positional_embeddings()
diff --git a/src/layers/whisper.cc b/src/layers/whisper.cc
index 8942e9f1..c8c8128d 100644
--- a/src/layers/whisper.cc
+++ b/src/layers/whisper.cc
@@ -13,9 +13,9 @@ namespace ctranslate2 {
                                                                  scope + "/layer",
                                                                  _num_heads,
                                                                  /*pre_norm=*/true,
-                                                                 ops::ActivationType::GELU,
+                                                                 ops::ActivationType::GELU
 #ifdef CT2_USE_HIP
-                                                                 model.use_flash_attention(),
+                                                                 , model.use_flash_attention()
 #endif
                                                                 ))
       , _output_norm(model, scope + "/layer_norm")
